src/index.ts

```diff
@@ -1,36 +1,101 @@
-import "reflect-metadata"
-import { AppDataSource } from "./data-source"
-import { User } from "./entity/User"
+import 'reflect-metadata';
 import { ApolloServer } from '@apollo/server';
 import { startStandaloneServer } from '@apollo/server/standalone';
+import { AppDataSource } from './data-source';
+import { User } from './entity/User';
+import 'bcrypt';
 
-AppDataSource.initialize().then(async () => {
-
-    const user = new User()
-    user.firstName = "3"
-    user.lastName = ""
-    user.age = 20
-
-    await AppDataSource.manager.save(user)
-})
-
+//GraphQl Schema
+//para colocar input (input NomeInput)
 const typeDefs = `#graphql
-    type Query {
+
+  type Query{
     hello: String
+  }
+
+  #input obrigatório !
+  type Mutation{
+    CreateUser(input: CreateUserInput!): User
```

Comment:
geralmente utilizamos `camelCase` para nomes de mutations
```suggestion
  type Mutation{
    createUser(input: CreateUserInput!): User
```